### PR TITLE
feat: [P4PU-539] removed transaction detail

### PR DESCRIPTION
**Pull request by oleksiybozhykntt**
#### List of Changes
Removed transaction detail

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: openapi/pagopa-arc-be.openapi.yaml [hunk 1]
```diff
@@ -683,16 +635,6 @@ components:
...
-      type: object
-      properties:
-        infoTransaction:
-          $ref: "#/components/schemas/InfoTransactionDTO"
-        carts:
-          type: array
-          items:
-            $ref: "#/components/schemas/CartItemDTO"
-          x-field-extra-annotation: "@lombok.Builder.Default"
     InfoTransactionDTO:
```
Comment (Giuseppe-LaManna) [inline openapi/pagopa-arc-be.openapi.yaml:638]: Is InfoTransactionDTO still used? if not, remove it

File: src/main/java/it/gov/pagopa/arc/exception/ArcExceptionHandler.java [hunk 2]
```diff
@@ -23,11 +23,6 @@ public ResponseEntity<ErrorDTO> handleBizEventsInvocationException(RuntimeExcept
         return handleArcErrorException(ex, request, HttpStatus.INTERNAL_SERVER_ERROR, ErrorDTO.ErrorEnum.GENERIC_ERROR);
     }
 
-    @ExceptionHandler(BizEventsTransactionNotFoundException.class)
-    public ResponseEntity<ErrorDTO> handleBizEventsTransactionNotFoundException(RuntimeException ex, HttpServletRequest request){
-        return handleArcErrorException(ex, request, HttpStatus.NOT_FOUND, ErrorDTO.ErrorEnum.TRANSACTION_NOT_FOUND_ERROR);
```
Comment (Giuseppe-LaManna) [inline src/main/java/it/gov/pagopa/arc/exception/ArcExceptionHandler.java:28]: Remove the ErrorEnum "transaction_not_found_error" from the swagger

**Review by Giuseppe-LaManna — CHANGES_REQUESTED** [review]

**Review by antonioT90 — APPROVED** [review]

File: openapi/pagopa-arc-be.openapi.yaml (hunk 1 above)
Comment (oleksiybozhykntt) [reply openapi/pagopa-arc-be.openapi.yaml]: removed

File: src/main/java/it/gov/pagopa/arc/exception/ArcExceptionHandler.java (hunk 2 above)
Comment (oleksiybozhykntt) [reply src/main/java/it/gov/pagopa/arc/exception/ArcExceptionHandler.java:28]: removed

**Review by Giuseppe-LaManna — APPROVED** [review]

**Pull request merged**